packages/earth-admin/src/components/organizations/organization-edit/OrganizationEdit.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { useState, useContext } from 'react';
+import { OrganizationEditProps } from '../model';
+import { useForm, Controller } from 'react-hook-form';
+import { LinkWithOrg } from 'components/LinkWithOrg';
+import { SearchInput } from 'components/search-input';
+import { handleOrganizationForm } from 'services/organizations';
+import { Auth0Context } from 'utils/contexts';
+import { navigate } from 'gatsby';
+import { ErrorMessages } from 'components/error-messages';
+
+const INPUT_SIZE_CLASSNAME = 'ng-width-1-1 ng-form-large';
+
+export default function OrganizationEdit(props: OrganizationEditProps) {
+  const {
+    data: { name, description, id },
+    newOrg,
+  } = props;
+
+  const { handleSubmit, register, errors, control, getValues, formState } = useForm({
+    mode: 'onChange',
+  });
+  const [serverErrors, setServerErrors] = useState(null);
+  const { selectedGroup } = useContext(Auth0Context);
+
+  const onSubmit = async (values: any) => {
+    const formData = getValues();
+
+    try {
+      await handleOrganizationForm(false, { description: formData.description }, id || formData.id, selectedGroup);
```

Comment:
use `updateOrganization(...)` directly instead of this function. See my comment near the function declaration.